resources/install.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Author:: Thom May (<thom@chef.io>)
+#
+# Copyright:: Copyright (c) 2016 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+include HabitatCookbook::Helpers
+
+resource_name :hab_install
+provides :hab_install
+
+property :install_url, String, default: "https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh"
+
+default_action :install
+allowed_actions :install, :upgrade
+
+action :install do
+  return if ::File.exist?(hab_path)
+
+  remote_file ::File.join(Chef::Config[:file_cache_path], "/hab-install.sh") do
```

Comment:
if we're doing a file join do we need this to be /hab-install.sh vs. hab-install.sh?